pkg/lib/cockpit-components-dropdown.tsx

```diff
@@ -35,49 +34,110 @@ import { EllipsisVIcon } from '@patternfly/react-icons';
  * This component expects a list of (non-deprecated!) DropdownItem's, if you
  * require a separator between DropdownItem's use PatternFly's Divivder
  * component.
+ *
+ * This component also supports DrilldownMenus. You have to follow these rules:
+ *
+ * - "containsDrilldown" must be true
+ * - "id" must be given, and all DrilldownMenus also must have "id"s.
+ * - The MenuItems that navigate between the drilldows (i.e. the ones
+ *   with a "direction" attribute) must have a itemId that starts with
+ *   "drilldown:", and no other MenuItem must have a itemId that
+ *   starts with "drilldown:".
  */
-export const KebabDropdown = ({ dropdownItems, position = "end", isDisabled = false, toggleButtonId, isOpen, setIsOpen } : {
+export const KebabDropdown = ({ dropdownItems, position = "end", popperProps, isDisabled = false, toggleButtonId, id, isOpen, setIsOpen, containsDrilldown = false } : {
```

Comment:
Nothing seems to override popperProps why do we need to be able to configure them?

Reply:
I needed that [here](https://github.com/cockpit-project/cockpit-machines/pull/2015/commits/86f5f21ea8b435fd1f5f8b1b229d593d7a229677#diff-5e76448f141d60170b26c4e0e17eb0a8f77a6ac7826c0e0807ca09a4119d6d1cR378)